.releaserc.json

```
@@ -32,6 +32,27 @@
         "assets": ["pyproject.toml", "docs/release_notes.md", "setup.py"],
```

```suggestion
        "assets": ["pyproject.toml", "docs/release_notes.md", "setup.py", "ibis/__init__.py"],
```

This change is necessary for semantic-release to commit the changes from the replace plugin.